presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improved performance of sum and average aggregations on decimals.
+* Support for LocationProvider supplied paths in Iceberg connector.
+* Column comments now available in JDBC connector.
+* Hidden columns ``file_modified_time`` and ``file_size`` added to Hive connector.
+* Partition lease support added to Hive connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix the Aggregation IF to FILTER rule by not unwrapping the IF expression inside the Aggregation. The rule now rewrites ``AGG(IF(condition, expr))`` to ``AGG(IF(condition, expr)) FILTER (WHERE condition)``.
+* Fix marking coordinator active so it is only done when the required minimum resource managers and required minimum coordinators are available in a cluster. Required resource managers can be configured by ``cluster.required-resource-managers-active`` (default value: 1, disaggregated coordinator only). Required coordinators can be configured by ``cluster.required-coordinators-active`` (default value: 1).
+* Improve performance of sum and average aggregations on decimals.
+* Add additional runtime stats for ``ScanFilterAndProject`` operator. Previously only ``TableScanOperator`` was exposed.
```

Comment:
Can we highlight what stats we have?

Reply:
This actually doesn't need to be in release notes. Release notes should be user facing features only.

Reply:
These statistics are user visible? If it's user visible and they would use it to debug their queries shouldn't we be mentioning something was added they can use?

Reply:
This particular PR doesn't add new stats. It just exposes existing stats for more query patterns. Maybe we can remove this item and merge it with the the metadata reading one below which added more counters.

Reply:
Removed. Thanks!